doc/jsk_topic_tools/scripts/static_tf_republisher.rst

```diff
@@ -0,0 +1,23 @@
+static_transform_republisher.py
+=============================
+
+
+What is this?
+-------------
+
+
+Node to republish /tf_static from a rosbag file
+
+
+Usage
+-----
+
+```
+rosrun jsk_topic_tools static_transform_republisher.py _file:=<absolute file path to a rosbag file>
+```
+
+or
+
+```
+rosrun jsk_topic_tools static_transform_republisher.py <file path to a rosbag file>
+```
```

Comment:
can you add document about parameter?
maybe this node only has `~file` parameter.

Reply:
Added